### ISSUE TITLE
0.8.018 sources do not contain the imagepack code for the new panty fap images, nor the images themselves.

### ISSUE THREAD

**Issue by dragontamer8740** [OXOIndustries/TiTS-Public#220]
0.8.018 sources do not contain the imagepack code for the new panty fap images, nor the images themselves.
I'm guessing that was added after pushing to public at the last moment?

**Comment by Fenoxo** [comment]
If I'm reading this right, there's not actually an issue, and this was just a complaint wanting the source repo updated?

**Comment by dragontamer8740** [comment]
No; at the time, the source code that was pushed did not match the published binaries in that they were lacking the assets of the image pack as well as the code for the assets.

I would still like to see the sources updated, of course, but this issue was different at the time.
